azure/packages/azure-local-service/package.json

```diff
@@ -50,6 +50,7 @@
 	},
 	"typeValidation": {
 		"disabled": true,
-		"broken": {}
+		"broken": {},
+		"entrypoint": "legacy"
```

Comment:
Does disabled=true, entrypoint=legacy mean something different than disabled=true, entrypoint=public?  I see both in this PR.

Reply:
They only differ if disabled becomes false/is removed. Then the entrypoints defined there become the default entrypoint for the type tests in that package.